src/api/network/asio/Imp.cpp

```diff
@@ -377,26 +355,22 @@ auto Asio::Imp::Receive(
         bufData.second,
         [this, connection{space(id)}, type, bufData, address{endpoint.str()}](
```

Comment:
This lambda appears to have the potential to be executed after the instance of Asio::Imp has been deleted, thus posing a crash hazard - do we need a new Jira ticket to capture the issue for careful examination?

Reply:
https://matterfi.atlassian.net/browse/MT-91
created